requirements.txt

```diff
@@ -1 +1,4 @@
-django>=1.7
+django>=1.7,<1.8
+django-extensions>=1.5,<1.6
```

Comment:
This is just many useful utilities that can be helpful for all of us.

You can now use the following command:
- `python manage.py shell_plus`
- `python manage.py shell_plus --print-sql`
- `python manage.py runserver_plus`

More at http://django-extensions.readthedocs.org/en/latest/command_extensions.html